lib/ruby/truffle/truffle/bigdecimal.rb

```diff
@@ -1,5 +1,15 @@
 # TODO require bigdecimal/*
```

Comment:
To go through https://github.com/pitr-ch/jruby/tree/bigdecimal/lib/ruby/stdlib/bigdecimal what should be loaded with `bigdecimal`. Doing it now to remove the todo.